data/json/vehicle_parts.json

```diff
@@ -2045,6 +2045,40 @@
         "breaks_into" : []
     },{
         "type" : "vehicle_part",
+        "id" : "chainsaw",
+        "name" : "chainsaw",
+        "symbol" : "c",
+        "color" : "red",
+        "broken_symbol" : "~",
+        "broken_color" : "red",
+        "damage_modifier" : 110,
+        "durability" : 275,
+        "item" : "chainsaw_off",
+        "difficulty" : 4,
+        "power": 600,
```

Comment:
It can be non-standard way of using ```power``` variable, I'm open for discussion, probably it's better to convert it to ```epower```.